addons/web/controllers/session.py

```diff
@@ -26,25 +28,34 @@ def get_session_info(self):
         request.session.touch()
         return request.env['ir.http'].session_info()
 
-    @http.route('/web/session/authenticate', type='json', auth="none")
+    @http.route('/web/session/authenticate', type='json', auth="none", readonly=False)
     def authenticate(self, db, login, password, base_location=None):
         if request.db and request.db != db:
             request.env.cr.close()
         elif request.db:
             request.env.cr.rollback()
```

Comment:
is this rollback still necessary?

Reply:
If runbot is green it is a good sign
The main problem was that authenticate would open a read write cursor during a readonly transaction because of the auth none.

I see that now the route is `readonly=False` so it should be ok. Actually the part over it may also be useless 

But @Julien00859 was the one that convinced me that having  `readonly=False`  on a `auth='none'` was a bad idea but I don't remember exactly why. Maybe it does not apply here.

Reply:
I don't remember either, let's do it.